.gitignore

```diff
@@ -1,4 +1,5 @@
 .vscode
+compile_commands.json
```

Comment:
what is this file ?

Reply:
File exported by cmake. Needed for some indexing tools(clangd for example). It should be in the workspace to work properly